### DISCUSSION TITLE
Black corner on dialog boxes in some apps.

### DISCUSSION THREAD

**Discussion by iamyegor** [i3/i3#5776] [Q&A]
In some applications like Thunar, Evince pdf reader I have this very ugly black borders on dialog boxes. I'm using [macos theme for linux](https://github.com/vinceliuice/WhiteSur-gtk-theme) and picom compositor for transparency and fading effects.
![Ugly black corners in i3](https://github.com/i3/i3/assets/102429874/f0b648c8-fe19-4a72-a66d-719e146166ae)

**Comment by okraits** [comment]
Does it look like that without picom as well? i3 doesn't render the content of windows, that's up to the UI toolkits.

**Reply by iamyegor** [reply]
Yes, disabling picom doen't help (((

**Reply by okraits** [reply]
Does it look like that with the default Adwaita theme as well?
